### PR TITLE
Clickable link

### PR DESCRIPTION
**Pull request by linhnhifm**

### CODE REVIEW

File: web/Web/web/static/js/document-viewer.js [hunk 1]
```diff
@@ -1064,6 +1067,16 @@ docView.prototype = {
       return "";
     }
 
+    function checkURL(str) {
```
Comment (ammsa) [inline web/Web/web/static/js/document-viewer.js:1070]: Minor edit. Change the name of this function to `isURL`

**Review by ammsa — CHANGES_REQUESTED** [review]
Just some minor edit. I haven't tested it but it looks good.

File: web/Web/web/static/js/document-viewer.js (hunk 1 above)
Comment (linhnhifm) [reply web/Web/web/static/js/document-viewer.js]: I just made the changes.

**Comment by ammsa** [comment]
The changes are now in production. Thanks.

**Comment by linhnhifm** [comment]
Thank you.

**Pull request merged**